src/components/site/SummaryList.js

```diff
@@ -1,20 +1,44 @@
 import React from "react";
+import PropTypes from "prop-types";
 
 const SummaryList = props => {
-  const { items = [] } = props;
+  const { id, items = [] } = props;
   return (
-    <div className="dg_summary-list">
+    <div id={id} key={id} className="dg_summary-list">
```

Comment:
i don't think the  `key` and `id` is need here, only inside the map?

Reply:
done